backend/__tests__/test_location/test_create_location.js

```diff
@@ -0,0 +1,64 @@
+const { createLocation, create } = require('../../routes/location/create_location')
+const { createLunchspace } = require('../../routes/lunchspace/create_lunchspace')
+const { createMockDatabase, dropMockDatabase } = require('../../lib/database/mock')
+const { mockReq, mockRes } = require('../../lib/express_mock')
+const { pool } = require('../../lib/database')
+
+const testName = 'McBurger'
+const testName2 = ''
+const testCoordinates = { lat: 20, long: 10 }
+const testCoordinates2 = 'test'
+const testLunchspaceId = 1
+
+describe('create location', () => {
+  beforeAll(createMockDatabase)
+  afterAll(dropMockDatabase)
+  beforeAll(async () => {
+    const setUpRequest = {
+      body: { lunchspaceName: 'Test Space', lunchspaceSubdomain: 'test-space' },
+      token: { userId: 1 },
+    }
+    const req = mockReq(setUpRequest)
+    const res = mockRes()
+    await pool.execute('INSERT INTO user (first_name, last_name)' +
+      'VALUES (?, ?)', ['Max', 'Mustermann'])
+    await createLunchspace(req, res)
+  })
+  describe('create', async () => {
+    it('should create a location in DB', async () => {
+      await expect(create(testName, testCoordinates, testLunchspaceId)).resolves.not.toThrow()
+    })
+  })
+  describe('createLocation', async () => {
+    const request1 = {
```

Comment:
request(1|2|3) are not shared between different test.
- [] move definition into the test function